src/Cli.Tests/EndToEndTests.cs

```diff
@@ -52,7 +54,7 @@ public void TestCleanup()
     public Task TestInitForCosmosDBNoSql()
     {
         string[] args = { "init", "-c", TEST_RUNTIME_CONFIG_FILE, "--database-type", "cosmosdb_nosql",
-                          "--connection-string", "localhost:5000", "--cosmosdb_nosql-database",
+                          "--connection-string", TEST_ENV_CONN_STRING, "--cosmosdb_nosql-database",
```

Comment:
This test uses `Verify()` , are there verify.txt files that need updating due to `TEST_CONNECTION_STRING` representing an env_var key?

Reply:
Connection strings are ignored in verify tests to avoid accidentally committing credentials to the repo.